composer.json

```
@@ -25,13 +25,13 @@
         }
     ],
     "require": {
-        "php": "~7.0",
+        "php": "^7.1 || ^8.0",
         "psr/log": "~1.0",
         "react/event-loop": "^1.0 || ^0.5 || ^0.4",
         "react/promise": "~2.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "~6.5"
+        "phpunit/phpunit": "^6.5 || ^7.5 || ^8.5 || ^9.3"
```

```suggestion
        "phpunit/phpunit": "^8.5 || ^9.3"
```

@edudobay I think only these two make sense for `"php": "^7.1 || ^8.0",`: https://phpunit.de/supported-versions.html

---

@standa If PHP 7.1 is still to be supported, I think we need PHPUnit 7 too; Composer won't let us install PHPUnit 8 with PHP 7.1.

---

I've dropped PHPUnit 6 and pushed a new commit to fix PHP 8 build. What do you think @standa?